Widget/app/src/test/java/com/microsoft/device/display/samples/widget/FetchRssFeedTest.kt

```diff
@@ -0,0 +1,99 @@
+package com.microsoft.device.display.samples.widget
+
+import android.content.Context
+import com.microsoft.device.display.samples.widget.feed.BaseSimpleApi
+import com.microsoft.device.display.samples.widget.feed.RssFeed
+import com.microsoft.device.display.samples.widget.feed.RssFeed.REQUEST_HEADER_VALUE
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.mockkClass
+import io.mockk.mockkObject
+import io.mockk.verify
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import retrofit2.Call
+import retrofit2.http.GET
+
+class FetchRssFeedTest {
+
+    private val mockContext = mockkClass(Context::class)
+    private val mockWebServer = MockWebServer()
+
+    init {
+        mockkObject(RssFeed)
+    }
+
+    @After
+    fun resetMocks() {
+        clearAllMocks()
+    }
+
+    @Test
+    fun fetchRssFeed_shouldCallParseXmlWithResponse_whenIsSuccessful() {
```

Comment:
In unit tests we can use single quote-format to name our tests so they can be read easier:
```
@Test
fun `should test that the string is 8 char lenght` () {}
```

Reply:
I would've liked to change it to single quote-format, it looks much more readable. But I've tried it and Android doesn't support having spaces in method names until API version 30, which means we cannot use it yet sadly.

This is from [Kotlin Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html):

> **Names for test methods**
> In tests (and only in tests), it's acceptable to use method names with spaces enclosed in backticks. (Note that such method names are currently not supported by the Android runtime.) Underscores in method names are also allowed in test code.

Reply:
Oh! forgot to add that I was just referring to **unit tests**. Exactly as you say for `instrumentation tests` this is till not supported.
This class if I'm not wrong is a unit test class, so you could do it here (and in others). Your call here ;-)

Reply:
Oh thank you for the clarification! 

Yes you are right, it only works for unit tests, but since we ended up having only 3 unit tests, I will go for keeping all the names without the single-quote format for consistency with the instrumentation ones. I will definitely use this in the future though!